cse-lib/network-processing/src/test/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangeProcessorTest.java

```diff
@@ -0,0 +1,189 @@
+package com.farao_community.farao.cse.network_processing.busbar_change;
+
+import com.farao_community.farao.cse.network_processing.TestUtils;
+import com.farao_community.farao.data.crac_creation.creator.cse.parameters.BusBarChangeSwitches;
+import com.farao_community.farao.data.crac_creation.creator.cse.parameters.SwitchPairId;
+import com.powsybl.computation.local.LocalComputationManager;
+import com.powsybl.iidm.export.Exporters;
+import com.powsybl.iidm.import_.ImportConfig;
+import com.powsybl.iidm.import_.Importers;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.TieLine;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BusBarChangeProcessorTest {
+
+    private Network network;
+    private Set<BusBarChangeSwitches> busBarChangeSwitchesSet;
+
+    private void setUp(String networkFile, String cracFile) {
+        network = Importers.loadNetwork(networkFile, getClass().getResourceAsStream("/BusBarChange/" + networkFile), LocalComputationManager.getDefault(), new ImportConfig(), null);
```

Comment:
It would be cleaner to recreate package path in the resources com/farao_community/farao/cse/network_processing/busbar_change and place all the files there. Then you could access directly by getResourceAsStream(networkFile)

Reply:
yes